test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smtrcomp.py

```diff
@@ -188,7 +188,7 @@ def test_smdp(self, sagemaker_session, ecr_image, framework_version, instance_co
         captured = capsys.readouterr()
         _assert_training_compiler_invoked(captured)
 
-
+    @pytest.mark.usefixtures("feature_smmp_present")
```

Comment:
Please leave 2 lines of space before the start of a function.

```suggestion

    @pytest.mark.usefixtures("feature_smmp_present")
```